### PR TITLE
Hotfix rsvp merge

### PR DESCRIPTION
**Pull request by YashoSharma**
-[x] Fixes improper schema migrations.

### CODE REVIEW

File: database/migration/V20160712_1603__createMailingListTables.sql [hunk 1]
```diff
@@ -26,7 +26,9 @@ INSERT INTO `mailing_lists` (`name`) VALUES ('idlers');
 INSERT INTO `mailing_lists` (`name`) VALUES ('applicants');
 INSERT INTO `mailing_lists` (`name`) VALUES ('accepted');
 INSERT INTO `mailing_lists` (`name`) VALUES ('waitlisted');
-INSERT INTO `mailing_lists` (`name`) VALUES ('attendees');
+INSERT INTO `mailing_lists` (`name`) VALUES ('software');
```
Comment (nmagerko) [inline database/migration/V20160712_1603__createMailingListTables.sql]: It looks like there are still some additions in here. Are there supposed to be?

Comment (YashoSharma) [inline database/migration/V20160712_1603__createMailingListTables.sql]: Yeah, we originally had those three mailing lists which we opted to replace with just the attendees one.

Comment (nmagerko) [inline database/migration/V20160712_1603__createMailingListTables.sql]: Ah okay. So the additions are just replacing what was there, correct?

Comment (YashoSharma) [inline database/migration/V20160712_1603__createMailingListTables.sql]: Yeah

**Pull request merged**